app/Http/Controllers/Auth/VerifyEmailController.php

```diff
@@ -15,13 +15,13 @@ class VerifyEmailController extends Controller
     public function __invoke(EmailVerificationRequest $request): RedirectResponse
     {
         if ($request->user()->hasVerifiedEmail()) {
-            return redirect()->intended(route('dashboard', absolute: false).'?verified=1');
+            return redirect()->intended(route('employee', absolute: false).'?verified=1');
```

Comment:
please revert this to dashboard. we have implemented a dashboard for common purposes not only for employee

Reply:
Reverted all changes done to Dashboard... create a new issue to properly implement the dashboard